lib/Phactory/Utils.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Phactory;
+
+class Utils {
+  public static function getValueOrReturn($value) {
+    if(is_object($value) && is_callable($value)
```

Comment:
fair enough.

On Thu, Jun 25, 2015 at 8:56 AM, Talysson de Oliveira Cassiano <
notifications@github.com> wrote:

> In lib/Phactory/Utils.php
> <https://github.com/chriskite/phactory/pull/42#discussion_r33256869>:
>
> > @@ -0,0 +1,13 @@
> > +<?php
> > +
> > +namespace Phactory;
> > +
> > +class Utils {
> > +  public static function getValueOrReturn($value) {
> > +    if(is_object($value) && is_callable($value)
>
> It is because, if I pass the name of a know function ('count', for
> example), is_callable would return true, but there's no way to know if
> user wanted the function count or the string 'count'.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chriskite/phactory/pull/42/files#r33256869>.
>